test_conformance/math_brute_force/function_list.cpp

```diff
@@ -274,96 +277,97 @@ const Func functionList[] = {
       0.0f,
       0.0f,
       0.0f,
+	  0.0f,
```

Comment:
Please ensure you follow the local indentation style.  Further down there are a few more inconsistent indents.
(clang-format is disabled for this table).

Reply:
Done